### PR TITLE
Sector 2 left half updates

### PR DESCRIPTION
**Pull request by JaggerTSG**

### CODE REVIEW

File: randovania/games/fusion/logic_database/Sector 2 TRO.txt [hunk 1]
```diff
@@ -233,16 +233,40 @@ Extra - room_id: [9]
...
-      Can Use Any Bombs
+      All of the following:
+          Morph Ball
+          Any of the following:
+              Morph Ball Bombs
+              All of the following:
+                  Knowledge (Advanced) and Normal Damage ≥ 88
+                  Any of the following:
+                      No Plasma Beam
+                      Diffusion Missiles and No Ice Missiles and No Missile Launcher and No Super Missiles
```
Comment (JeffGainsNGames) [inline randovania/games/fusion/logic_database/Sector 2 TRO.txt:248]: There will be an option to turn off items so there is no need to have `no item` requirement.

The generator HATES these statements. Please remove and avoid these in the future.

File: randovania/games/fusion/logic_database/Sector 2 TRO.txt [hunk 2]
```diff
@@ -233,16 +233,40 @@ Extra - room_id: [9]
...
+              Wall Jump (Beginner) and Can Freeze Enemies
   > Door to Maintenance Annex
-      Can Use Any Bombs
+      All of the following:
+          Morph Ball
+          Any of the following:
+              Morph Ball Bombs
+              All of the following:
+                  Knowledge (Advanced) and Normal Damage ≥ 88
+                  Any of the following:
```
Comment (JeffGainsNGames) [inline randovania/games/fusion/logic_database/Sector 2 TRO.txt:246]: Assuming this is some sort of clip? (my knowledge is clearly not advanced)

File: randovania/games/fusion/logic_database/Sector 2 TRO.txt [hunk 3]
```diff
@@ -233,16 +233,40 @@ Extra - room_id: [9]
...
+                  Space Jump
+                  Any of the following:
+                      Power Bombs or Screw Attack
+                      Morph Ball Bombs and Jump Bombjump (Expert)
+              All of the following:
+                  High Jump
+                  Any of the following:
+                      Power Bombs
+                      Morph Ball Bombs and Jump Bombjump (Advanced)
+                      Screw Attack and Wall Jump (Beginner)
```
Comment (JeffGainsNGames) [inline randovania/games/fusion/logic_database/Sector 2 TRO.txt:269]: I'd probably bump this to intermediate wall jump.

File: randovania/games/fusion/logic_database/Sector 2 TRO.txt [hunk 4]
```diff
@@ -873,7 +913,13 @@ Extra - room_id: [31, 7]
...
       Trivial
   > Door to Data Save Room
-      Can Use Any Bombs
+      Any of the following:
+          Can Use Any Bombs
+          All of the following:
+              Morph Ball
+              Any of the following:
+                  Screw Attack
+                  Speed Booster and Movement (Intermediate)
```
Comment (JeffGainsNGames) [inline randovania/games/fusion/logic_database/Sector 2 TRO.txt:922]: Morph blocks break from speedbooster? this should probably have knowledge tied to it as well.

File: randovania/games/fusion/logic_database/Sector 2 TRO.txt [hunk 5]
```diff
@@ -1273,11 +1319,19 @@ Extra - room_id: [54]
...
+                  Speed Booster and Movement (Expert) and Shinespark Tricks (Advanced)
   > Pickup (Power Bomb Tank)
       Any of the following:
           Space Jump
-          Speed Booster and Shinespark Tricks (Advanced)
+          All of the following:
+              Wall Jump (Intermediate)
+              Any of the following:
+                  Movement (Hypermode)
+                  Speed Booster and Movement (Expert) and Shinespark Tricks (Advanced)
```
Comment (JeffGainsNGames) [inline randovania/games/fusion/logic_database/Sector 2 TRO.txt:1334]: any chance you know of a video for this? lol

**Review by JeffGainsNGames — CHANGES_REQUESTED** [review]

**Comment by JeffGainsNGames** [comment]
I rewrote and pushed the changes requested, this can be closed. Also note please make commit titles more descriptive than  `i did stuffs` in the future.

File: randovania/games/fusion/logic_database/Sector 2 TRO.txt (hunk 1 above)
Comment (JaggerTSG) [reply randovania/games/fusion/logic_database/Sector 2 TRO.txt:248]: removing.

File: randovania/games/fusion/logic_database/Sector 2 TRO.txt (hunk 2 above)
Comment (JaggerTSG) [reply randovania/games/fusion/logic_database/Sector 2 TRO.txt:246]: it's a damage boost, and repeated pausing to reduce iframes to get another damage boost. One boost isn't enough height to get pushed over that tile in the first tunnel. Shark said that enemy deals 44 damage.

File: randovania/games/fusion/logic_database/Sector 2 TRO.txt (hunk 3 above)
Comment (JaggerTSG) [reply randovania/games/fusion/logic_database/Sector 2 TRO.txt:269]: changing.

File: randovania/games/fusion/logic_database/Sector 2 TRO.txt (hunk 4 above)
Comment (JaggerTSG) [reply randovania/games/fusion/logic_database/Sector 2 TRO.txt:922]: yup, so long as they're not chain blocks. But only if you collide into them. I don't think they break if you run across the top of them. Should that be Intermediate? That's my guess.

File: randovania/games/fusion/logic_database/Sector 2 TRO.txt (hunk 2 above)
Comment (JaggerTSG) [reply randovania/games/fusion/logic_database/Sector 2 TRO.txt:246]: https://www.twitch.tv/jaggerg/clip/JazzyStupidFloofBigBrother-Uo0pmt-pKydPhdcv?filter=clips&range=all&sort=time

**Pull request closed without merging**